rpc_jobs/standard_job_premerge.yml

```diff
@@ -94,6 +95,7 @@
         env.RE_JOB_ACTION = "{action}"
         env.RE_JOB_FLAVOR = "{FLAVOR}"
         env.RE_JOB_REPO_NAME = "{repo_name}"
+        env.BOOT_TIMEOUT = "{BOOT_TIMEOUT}"
```

Comment:
This variable doesn't match with the comment although it does get passed through. Looking more, not all the RE_ variables are collected here anyway so maybe it doesn't matter.

Reply:
Can you clarify which comment you're referring to?  The only reason why I have to export it here is because it gets used subsequently by ansible.

Reply:
```
         // Pass details about the job parameters through
         // to the target environment so that scripts can
         // use them to adapt behaviour.
```
It's tacked onto the end of a block for which the comment is indicating they are there for the benefit of gating scripts which this is not.